index.html

```diff
@@ -20,7 +20,7 @@ <h2 class="greetings">Glad to meet you!</h2>
                 <p>If you will like what you see and have a project you will need coded,don't hesitate to contact me</p> 
                 <p>look through some of my work and experince</p>
             </div>
-            <h3 class="heading" style="color:blue;">LET'S CONNECT</h3>
+            <h3 class="heading">LET'S CONNECT</h3>
             <div class="icon-bar">
                 <img src="/home/ignatius/Documents/Microverse Projects/Portfolio/images/twitter.png" alt="">
                 <img src="/home/ignatius/Documents/Microverse Projects/Portfolio/images/linkedin.png" alt="">
```

Comment:
- [ ] kindly consider using a `ul` tag and then `li`s tags and in each `li`, consider taking `a` tag and add your each icon in it as suggested by the previous three reviewers